ambry-tools/src/main/java/com/github/ambry/clustermap/HelixTaskWorkflowManagerTool.java

```diff
@@ -108,6 +112,10 @@
     @Default("false")
     final boolean deleteSpecifiedWorkflow;
 
+    @Config("task.type")
+    @Default("AGGREGATION_TASK")
```

Comment:
Should we use AGGREGATE_TASK to align with "TaskType.AGGREGATE_TASK.name()"?

Reply:
fixed.